package.json

```diff
@@ -57,9 +57,8 @@
     "test:playground:build": "build-storybook",
     "test:playground:ci": "start-storybook -p 5000 -s public --ci",
     "test:types": "tsc",
-    "test:unit": "npm test",
+    "test:unit": "npm run test --",
```

Comment:
When do you need the `--` and when don't you. Perpetually confused.

Reply:
Me too, I had to play around with it until the coverage tests worked again.